app/app/urls.py

```diff
@@ -137,20 +137,16 @@
     url(r'^tip/?', dashboard.tip_views.send_tip, name='tip'),
 
     # Legal
-    re_path(r'^terms/?', dashboard.views.terms, name='_terms'),
-    re_path(r'^legal/terms/?', dashboard.views.terms, name='terms'),
-    re_path(r'^legal/privacy/?', dashboard.views.privacy, name='privacy'),
-    re_path(r'^legal/cookie/?', dashboard.views.cookie, name='cookie'),
-    re_path(r'^legal/prirp/?', dashboard.views.prirp, name='prirp'),
-    re_path(r'^legal/apitos/?', dashboard.views.apitos, name='apitos'),
-    re_path(r'^legal/?', dashboard.views.terms, name='legal'),
+    re_path(r'^legal/?$', dashboard.views.terms, name='terms'),
+    re_path(r'^legal/terms/?$', dashboard.views.terms, name='terms'),
```

Comment:
That's what I had done initially but then @owocki said he would like both [here](https://github.com/gitcoinco/web/pull/1917#discussion_r208018634) ! hence added it back

Reply:
@thelostone-mc Okay, but can we rename the additional one to `legal_terms` or something?  If not, it might cause reverse collisions since we're registering both with the same name.

Reply:
Done